site/_posts/2022-11-01-authenticators-ceremonies-webauthn-oh-my.md

```diff
@@ -0,0 +1,128 @@
+
+WebAuthn provides a means for servers to authenticate users using cryptographic keys (private and public) in place of passwords. The private and public keys are used to authenticate users, and the public key is stored in the server (also known as a relying party or RP), while the private key is stored locally in the user's device.
+
+The server uses a credential id to identify the user. On the other hand, the user's device contains the private key that corresponds to the public key (which is stored in the server and is useless without a corresponding private key). 
+
+## Why You Need WebAuthn
+
+You might be wondering why you should use WebAuthn instead of using a more robust password from a password generator. Or you may be considering adding 2FA for more security, which is an improvement but still prone to cyberattacks (*ie* phishing) [compared to WebAuthn](https://www.slashid.dev/blog/webauthn-antiphishing#:~:text=WebAuthn%20Anti%2DPhishing,superior%20authentication%20method.).
+
+Here are a few of the reasons WebAuthn is the superior choice for keeping your application secure:
+
+* WebAuthn helps resolve significant security problems related to data breaches, phishing, and attacks against 2FA methods. Phishing attacks are usually disguised emails or messages luring users to click on malicious links with the intent of stealing sensitive data like login credentials. With WebAuthn, attackers can't gain access to your data using login credentials stolen from your device since WebAuthn uses cryptographic keys to authenticate users and the private key never leaves the device nor is accessible to the user.
+* It's easier to use since users have to manage fewer passwords.
+* It eliminates the need for servers to store passwords. With passwords, if a server happens to be compromised, the user might be at risk if the password wasn’t hashed properly. With WebAuthn, if the server is compromised, the server only stores the public key, which requires the private key (stored in the user's device) to complete the authentication.
+* It's easier for attackers to steal a user's password than to steal their device. Even if an attacker does steal a device, they still need to bypass the device's security (*ie* biometrics or facial recognition system).
+* WebAuthn is easy to implement because it consists of [browser APIs that can be easily used by developers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API#examples).
+* It offers a smooth authentication experience for users.
+
+## How WebAuthn Works
+
+WebAuthn consists of three components: the authenticator (the user's device), the web browser, and the server (RP). In essence, the user initiates login via the browser, the browser then prompts the user to authenticate using their device (via biometrics or facial recognition, for example), and then if successful, the user is logged into the web application.
```

Comment:
Facial recognition is a form of biometrics. I'd change this to be either:
* fingerprint or facial recognition
* biometrics or PIN